documentation/docs/README.md

```diff
@@ -0,0 +1 @@
+## Readme
```

Comment:
Should this have some descriptive text, or is it one of those required boilerplate files that never gets used?

Reply:
No this is just the beginning